docs/getting-started.md

```diff
@@ -286,14 +289,17 @@ Change handleClick to be:
 
 The data format we are returning here is based part of the ans [custom embed element](https://github.com/washingtonpost/ans-schema/blob/master/src/main/resources/schema/ans/0.10.3/story_elements/custom_embed.json) schema.
 
-Finally, we can test our whole flow by using the online custom embed testing tool. Visit the testing tool, select the config options, and set the search page to be http://localhost/pf/resources/composer/search.html.
+Finally, we can test our whole flow by using the online custom embed testing tool. Visit the testing tool, select the config options, and set the search page to be http://localhost/pf/resources/plugins/composer/search.html.
```

Comment:
the path should be /resources/plugins/composer/embeds/movie/search.html

Reply:
Oh right. My mistake.

Reply:
u still need the /pf btw